examples/karaf-rest-example/karaf-rest-example-client/src/main/java/org/apache/karaf/examples/rest/client/ListBean.java

```diff
@@ -0,0 +1,90 @@
+/*
+ *  Licensed to the Apache Software Foundation (ASF) under one or more
+ *  contributor license agreements.  See the NOTICE file distributed with
+ *  this work for additional information regarding copyright ownership.
+ *  The ASF licenses this file to You under the Apache License, Version 2.0
+ *  (the "License"); you may not use this file except in compliance with
+ *  the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.apache.karaf.examples.rest.client;
+
+import org.apache.karaf.examples.rest.Booking;
+import org.apache.karaf.examples.rest.BookingService;
+
+/**
+ * Simple bean controlling a thread to display the bookings on System.out.
+ */
+public class ListBean {
+
+    private BookingService bookingService;
```

Comment:
Maybe the client can be a REST client for this example (using the CXF REST BeanProxy for instance, or directly using the Java  HTTP client).

Reply:
We could have 2 examples, one simple with the Java HTTP client and one with Apache CXF inlcude logging interceptor (it could be usefull for developers when consuming REST service).

Reply:
It sounds good !